package.json

```diff
@@ -296,5 +296,8 @@
     "@vscode/codicons": "^0.0.36",
     "groq-sdk": "^0.7.0",
     "mermaid": "^11.3.0"
+  },
+  "overrides": {
+    "katex": "0.16.21"
```

Comment:
Could you run `npm update katex` to update the package in the lockfile instead? The override itself might become stale.

Reply:
yep, I didn't know that worked.